frontend/src/metabase/components/UserAvatar/UserAvatar.styled.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+import { color, height, width } from "styled-system";
+import { color as brandColor } from "metabase/lib/colors";
+
+export interface AvatarProps {
+  color?: string;
+  height?: string[];
+  width?: string[];
+  bg?: string;
+}
+
+export const Avatar = styled.div<AvatarProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 999px;
+  font-weight: 900;
+  line-height: 1;
+  ${height};
+  ${width};
+  ${color};
+  background-color: ${({ bg = brandColor("brand") }) => bg};
+`;
+Avatar.defaultProps = {
```

Comment:
`defaultProps` are discouraged and are deprecated for functional components https://github.com/facebook/react/pull/16210

Reply:
Is there an alternative for `defaultProps` for emotion components? I only use `defaultProps` for emotion otherwise, it's gonna require a wrapper for every emotion components that want to have default props.

Reply:
Probably something like `height: ${({ height = "3rem" }) => height}`;

Reply:
I could work with that.